### ISSUE TITLE
Add bucket_id calculation function

### ISSUE THREAD

**Issue by Totktonada** [tarantool/ddl#76]
After #71 we'll have all information that is necessary for calculating bucket_id by a sharding key. The proposal is to implement this helper function in the module.

## Example

Schema:

```
TBD
```

Assume we serve HTTP requests:

```lua
function handler(req)
    local bucket_id = ddl.bucket_id('users', req:json()['user_id'])
    local ok, err = vshard.router.callrw(bucket_id, <..args..>)
    <...>
end
```

The key point is that ddl knows, which sharding function is is use. Without the function we would do the following:

```lua
function handler(req)
    local bucket_id = vshard.router.bucket_id_mpcrc32(req:json()['user_id']) -- !!
    local ok, err = vshard.router.callrw(bucket_id, <..args..>)
    <...>
end
```

**Comment by ligurio** [comment]
Discussed verbally with @Totktonada:

- name of the new method is `bucket_id` (not `get_bucket_id`)
- user pass sharding key that is ready for use in `bucket_id`
- it should not be a default for hash function (in a future it would be nice to have it, but now with introducing the default hash function we may break DDL users)

**Comment by ligurio** [comment]
Blocked by #71
